### PR TITLE
Improvement: Inline notice - HTML Optimization.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- [Doc](https://docs.google.com/document/d/1ZNsQoG4eEtac9_bxWwwt6XBkOSdfarQ3D9vaulpcss0/edit)

### Screenshots

-  https://share.getcloudapp.com/Jrun22jw

### How has this been tested?

- Backward testing.
- All setting option.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by RajkiranBagal — COMMENTED** [review]
We can remove uagb-notice-text div and add it's class to child p tag

**Comment by AbhijitNage123** [comment]
> We can remove uagb-notice-text div and add it's class to child p tag

@RajkiranBagal  No we can't it shows an issue on the front-end as discussed on the call. See here - https://share.getcloudapp.com/DOu6ppJg So keeping it as it is Please check

File: assets/js/inline-notice.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ UAGBInlineNotice = { // eslint-disable-line no-undef
 			if ( 'undefined' === typeof currentCookie && true === isCookie ) {
 				mainWrap.style.display = 'block';
 			}
-			const closeBtn = mainWrap.querySelector( '.uagb-notice-dismiss' );
+			
+			const closeBtn = mainWrap.querySelector( '.uagb-notice-dismiss' ) ? mainWrap.querySelector( '.uagb-notice-dismiss' ) : mainWrap.querySelector( 'svg' );
```
Comment (RajkiranBagal) [inline assets/js/inline-notice.js:20]: Take mainWrap.querySelector( '.uagb-notice-dismiss' ) in a variable

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: assets/js/inline-notice.js (hunk 1 above)
Comment (AbhijitNage123) [reply assets/js/inline-notice.js]: Updated

**Review by RajkiranBagal — APPROVED** [review]

**Comment by akshaylakh** [comment]
working fine

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**